sample-queries/sample-queries.json

```diff
@@ -1608,16 +1608,16 @@
         "id": "ae8adc92-9f78-4065-b4b9-22572bc98492",
         "category": "Search (beta)",
         "method": "POST",
-        "humanName": "search messages",
-        "requestUrl": "/beta/search/query",
-        "docLink": "https://aka.ms/mssearchapi",
+        "humanName": "Search Messages",
```

Comment:
@lanhongvp the format for samples is to have the human name in all small letters.